src/sidebar/services/annotations-exporter.ts

```diff
@@ -83,14 +83,21 @@ export class AnnotationsExporter {
     const annotationsText = annotations
       .map((annotation, index) => {
         const page = pageLabel(annotation);
+        // Lines that should only be rendered if the element they represent exists
+        const conditionalLines = [
+          annotation.tags.length > 0
+            ? `Tags: ${annotation.tags.join(', ')}`
+            : undefined,
+          page ? `Page: ${page}` : undefined,
+        ].filter(Boolean);
```

Comment:
This approach works because all "conditional lines" are rendered sequentially.

We have considered rearranging lines, which probably means we have to define an array with all lines, but for now this works.